api/background/utils.ts

```diff
@@ -28,8 +28,16 @@ export class ApiUtils {
     });
   }
 
-  static async getSettings(): Promise<Settings | null> {
-    return await storage.getItem<Settings>(SETTINGS_KEY);
+  static async getSettings() {
+    return await storage.getItem<Settings>(SETTINGS_KEY, {
```

Comment:
Why fallback? I think it would be better to return `null` if there is no data.

Reply:
In case the settings have not been stored yet, I return the default values.
This is matching the `SettingsProvider` behavior.